packages/dialog/test/dialog.test.js

```diff
@@ -161,4 +162,28 @@ describe('vaadin-dialog', () => {
       expect(contentMinWidth).to.be.gt(0);
     });
   });
+
+  describe('focus restoration', () => {
+    let dialog, button;
+
+    beforeEach(() => {
+      dialog = fixtureSync('<vaadin-dialog></vaadin-dialog>');
+      button = fixtureSync('<button></button>');
```

Comment:
```suggestion
    let wrapper, dialog, button;

    beforeEach(() => {
      wrapper = fixtureSync(`
        <vaadin-dialog></vaadin-dialog>
        <button></button>
      `);
      [dialog, button] = wrapper.children;

Reply:
We generally wrap elements like that, so my previous suggestion was indeed wrong:

```
      wrapper = fixtureSync(`
        <div>
          <vaadin-dialog></vaadin-dialog>
          <button></button>
        </div>
      `);
```

IMO this coding style is self-explanatory, and we generally use it across our codebase.

Reply:
Done.